crates/primitives/src/transaction/mod.rs

```diff
@@ -1265,6 +1265,7 @@ impl TransactionSigned {
 
 impl SignedTransaction for TransactionSigned {
     type Type = TxType;
+    type Transaction = Transaction;
```

Comment:
this was removed because this AT, didn't make much sense that a transaction defines a transaction